code/modules/security_levels/keycard_authentication.dm

```diff
@@ -52,11 +52,14 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/keycard_auth, 26)
 	return data
 
 /obj/machinery/keycard_auth/ui_status(mob/user)
-	if(isanimal(user))
-		var/mob/living/simple_animal/A = user
-		if(!A.dextrous)
-			to_chat(user, span_warning("You are too primitive to use this device!"))
-			return UI_CLOSE
+	if(!isanimal(user))
+		return ..()
+	if(isdrone(user))
+		return UI_CLOSE
```

Comment:
You mind flipping these? I don't want this code failing if someone makes drones not simple mobs